test/built-ins/Array/prototype/toLocaleString/resizable-buffer.js

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.tolocalestring
+description: >
+  Array.p.toLocaleString behaves correctly on TypedArrays backed by resizable
+  buffers.
+includes: [resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+---*/
+
+for (let ctor of ctors) {
+  const rab = CreateResizableArrayBuffer(4 * ctor.BYTES_PER_ELEMENT, 8 * ctor.BYTES_PER_ELEMENT);
+  const fixedLength = new ctor(rab, 0, 4);
+  const fixedLengthWithOffset = new ctor(rab, 2 * ctor.BYTES_PER_ELEMENT, 2);
+  const lengthTracking = new ctor(rab, 0);
+  const lengthTrackingWithOffset = new ctor(rab, 2 * ctor.BYTES_PER_ELEMENT);
+  const taWrite = new ctor(rab);
+
+  // toLocaleString separator is implementation dependent.
+  function listToString(list) {
+    const comma = ['',''].toLocaleString();
+    const len = list.length;
+    let result = '';
+    if (len > 1) {
+      for (let i=0; i < len - 1 ; i++) {
+        result += list[i] + comma;
+      }
+    }
+    if (len > 0) {
+      result += list[len-1];
+    }
+    return result;
```

Comment:
There's an array method for this :smile: 
```suggestion
    return list.join(comma);
```